src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -445,26 +433,6 @@ public void ApplicationIdPlaceholder ()
 			}
 		}
 
-		[Test]
-		[Category ("XamarinBuildDownload")]
-		public void ExtraAaptManifest ()
```

Comment:
Why is this test removed?  It doesn't appear to (directly?) use the Support libraries.  Is there not a newer Crashytics?  Looks like there's a [Xamarin.Firebase.Crashlytics](https://www.nuget.org/packages/Xamarin.Firebase.Crashlytics) package which exists…

This test is also weird, in that it's the only test that uses `Crashlytics.HandleManagedExceptions()`…

Reply:
I was able to put it back, using:
* `Xamarin.Firebase.Crashlytics` NuGet
* `Firebase.Crashlytics.FirebaseCrashlytics.Instance.SendUnsentReports()`
* Look for `firebaseinitprovider` in the `AndroidManifest.xml`